News_Blockchain/Database.cs

```diff
@@ -135,11 +143,16 @@ public Block GetRecord(string key)
         return null;
     }
 
+    /// <summary>
+    /// Function returns a list of blocks from including specified index and up to latest block
+    /// </summary>
+    /// <param name="index"></param>
+    /// <returns></returns>
     public List<Block> GetLastSpecifiedBlocks(int index)
     {
         List<Block> list = new List<Block>();
         var iterator = _zoneTree.CreateIterator();
-        for (int i = 0; i < index; i++)
+        for (int i = index; i <= lastKnownBlockIndex; i++)
```

Comment:
Function functionality has been changed. @bajla Confirm if it was intended to work in this way